src/game/interactables_2/egg_utopia/launcher.c

```diff
@@ -147,32 +147,32 @@ void CreateEntity_Launcher(MapEntity *me, u16 spriteRegionX, u16 spriteRegionY,
             launcher->s.palId = 0;
 
             s->unk28->unk0 = -1;
-            s->unk10 = SPRITE_FLAG_PRIORITY(2);
+            s->unk10 = SPRITE_FLAG(PRIORITY, 2);
             s->graphics.dest = VramMalloc(EGG_UTO_LAUNCHER_TILE_COUNT);
             s->graphics.anim = SA2_ANIM_LAUNCHER_EGG_UTO;
             launcher->s.variant = 0;
 
 #ifndef NON_MATCHING
             // This is completely redundant because of the switch below.
             if (kind == LAUNCHER_KIND(LAUN_DIR_LEFT, LAUN_GRAVITY_DOWN)) {
-                s->unk10 |= (SPRITE_FLAG_X_FLIP(1) | SPRITE_FLAG_Y_FLIP(0));
+                s->unk10 |= (SPRITE_FLAG(X_FLIP, 1) | SPRITE_FLAG(Y_FLIP, 0));
```

Comment:
My only concern with this is that it's not possible to autocomplete the param1 input, as the editor has no idea of the possible values which can go here?